lib/cli.js

```diff
@@ -9,29 +9,36 @@ var yosay = require('yosay');
 var stringLength = require('string-length');
 var rootCheck = require('root-check');
 var meow = require('meow');
+var list = require('cli-list');
 var pkg = require('../package.json');
 var Router = require('./router');
 
-var cli = meow({
-  help: false,
-  pkg: pkg
-});
+var gens = list(process.argv.slice(2));
 
-var opts = cli.flags;
-var args = cli.input;
-var cmd = args[0];
-var insight;
+var cli = gens.map(function (gen) {
+  var minicli = meow({ help: false, pkg: pkg, argv: gen });
+  var opts = minicli.flags;
+  var args = minicli.args;
 
-// add un-camelized options too, for legacy
-// TODO: remove some time in the future when generators have upgraded
-Object.keys(opts).forEach(function (key) {
-  var legacyKey = key.replace(/[A-Z]/g, function (m) {
-    return '-' + m.toLowerCase();
+  // add un-camelized options too, for legacy
+  // TODO: remove some time in the future when generators have upgraded
+  Object.keys(opts).forEach(function (key) {
+    var legacyKey = key.replace(/[A-Z]/g, function (m) {
+      return '-' + m.toLowerCase();
+    });
+
+    opts[legacyKey] = opts[key];
   });
 
-  opts[legacyKey] = opts[key];
+  return { opts: opts, args: args };
 });
 
+var base = cli[0] || { opts: {}, args: {} };
+var opts = base.opts;
```

Comment:
Yeah, I think this would convey more meaning and make the code easier to reason about :)